### PR TITLE
Start refactoring Customers First

### PR DESCRIPTION
**Pull request by johnboxall**
I'm trying to slowly make this flow more like a article than a list of bullet points.

Cleans up some of our points around testing based on recent discussions.

Also moves "Follow Best Practices" to "Simply" which I believe is a better home for it.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -106,8 +117,11 @@ our company and culture evolve, so should this document.
 
 ### Invent
 
-#### Don't be afraid to push the boundaries of technology, but [be conservative with your tech stack](http://mcfunley.com/choose-boring-technology).
-- What this means is that the things you are building should be pushing boundaries, not the things you are building them with. [Here is a list of technology decisions we have evaluated in the past](https://mobify.atlassian.net/wiki/questions/77168927/what-should-i-consider-when-evaluating-a-new-service-feature-or-technology-what-criteria-do-we-use-in-our-decisions).
+#### Push the boundaries of technology, but [try to map new problems to our stack](http://mcfunley.com/choose-boring-technology).
+
+We're inventing a new product category. This is pretty tough.
+
+Favour battle-tested technologies that we're familiar with.
```
Comment (jeremywiebe) [inline README.md]: Should we add something to the effect of "Choose old tech for new problems and consider new tech for 1-to-n solutions."   Also, the "Favour battle-tested" line goes with the previous line, doesn't it?... why the extra line break... makes them feel unrelated. :)

File: README.md [hunk 2]
```diff
@@ -54,19 +54,30 @@ our company and culture evolve, so should this document.
...
+
+Testing leads to better code. *Nothing refactors a giant method better than
+trying to write a test for it!*
+
+Testing gives us the confidence to experiment and deploy to production. This
+is especially important with complex code like [`mobify/capturejs`](https://github.com/mobify/capturejs).
+A strong [test suite](https://github.com/mobify/capturejs/blob/master/tests/capture-tests.js)
+let's us make improvements without worrying about breaking edgecases.
+
+Favour writting tests when:
```
Comment (jeremywiebe) [inline README.md]: `writing`

**Comment by jansepar** [comment]
Hmm should have maybe looked at this one first - it now conflicts with some recent changes.
